graylog2-web-interface/src/components/common/ConfirmLeaveDialog.tsx

```diff
@@ -23,17 +22,20 @@ import AppConfig from 'util/AppConfig';
 
 /**
  * This component should be conditionally rendered if you have a form that is in a "dirty" state. It will confirm with the user that they want to navigate away, refresh, or in any way unload the component.
+ * The `ignoredRoutes` prop is an array of routes that should not trigger the confirmation dialog.
  */
 type Props = {
-  question: string,
+  question?: string,
+  ignoredRoutes?: Array<string>,
 };
 
-const locationHasChanged = (currentLocation: Location, newLocation: Location, question: string) => ((newLocation.pathname !== currentLocation.pathname)
-  // eslint-disable-next-line no-alert
-  ? !window.confirm(question)
-  : false);
+const locationHasChanged = (currentLocation: Location, newLocation: Location, question: string, ignoredRoutes: Array<string>) => (
+  (newLocation.pathname !== currentLocation.pathname && !ignoredRoutes.includes(newLocation.pathname))
+    // eslint-disable-next-line no-alert
+    ? !window.confirm(question)
```

Comment:
Not in scope but we could consider replacing it with our `<ConfirmDialog>` for consistent UI and getting rid of the eslint-disable. Maybe we can create an issue for that for the frontend backlog.